gulpfile.js

```diff
@@ -1,29 +1,54 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
-var gulp = require("gulp");
-var log = require('fancy-log');
-var istanbul = require('gulp-istanbul');
-var isparta = require('isparta');
-var sourcemaps = require("gulp-sourcemaps");
-var path = require("path");
-var preprocess = require("gulp-preprocess");
-var install = require("gulp-install");
-var runSequence = require("run-sequence");
-var ts = require("gulp-typescript");
-var mocha = require("gulp-mocha");
-var GulpExtras = require("./tools/gulp-extras");
-var minimist = require("minimist");
-var os = require("os");
-var fs = require("fs");
-var Q = require("q");
-var remapIstanbul = require('remap-istanbul/lib/gulpRemapIstanbul');
-var execSync = require('child_process').execSync;
-
-var copyright = GulpExtras.checkCopyright;
-var imports = GulpExtras.checkImports;
-var executeCommand = GulpExtras.executeCommand;
+const gulp = require("gulp");
+const log = require('fancy-log');
+const istanbul = require('gulp-istanbul');
+const isparta = require('isparta');
+const sourcemaps = require("gulp-sourcemaps");
+const path = require("path");
+const preprocess = require("gulp-preprocess");
+const install = require("gulp-install");
+const runSequence = require("run-sequence");
+const ts = require("gulp-typescript");
+const mocha = require("gulp-mocha");
+const GulpExtras = require("./tools/gulp-extras");
+const minimist = require("minimist");
+const os = require("os");
+const fs = require("fs");
+const Q = require("q");
+const es = require('event-stream');
```

Comment:
@SounD120 Seems to be, you forgot to add `event-stream` to devDependecies.

Reply:
Thanks for pointing out. I'll fix that.